platform_versioned_docs/version-24.2/compute-envs/azure-batch.mdx

```diff
@@ -144,15 +144,21 @@ To create an access key:
     - Add the **Batch account** and **Blob Storage account** names and access keys to the relevant fields.
 1. Delete the copied keys from their temporary location after they have been added to a credential in Platform.
 
-#### Entra service principal
+#### Entra service principal and managed identity
+
+If using Entra for authentication, you must create a service principal and managed identity. Seqera Platform uses the Service Principal to authenticate to Azure Batch and Azure Storage. It submits a Nextflow task as the head process to run Nextflow, which authenticates to Azure Batch and storage using the Managed Identity attached to the node pool.
+
+Therefore, you must create both an Entra service principal and a managed identity. You add the service principal to your Seqera Platform credentials and attach the managed identity to your Azure Batch node pool which will run Nextflow.
```

Comment:
If I understand this correctly, two different MSFT identities need to be used:

1. Service Principle type -- which can accommodate the fact that Tower lives outside the Azure network and likely is calling across the public internet.

2. Managed Identity -- can be used since the Pool lives inside the account and thus has a greater assurance level.

Correct?

Yes, explained below:
> When you use a manually configured compute environment with a managed identity attached to the Azure Batch Pool, Nextflow can use this managed identity for authentication. However, Platform still needs to use access keys or an Entra service principal to submit the initial task to Azure Batch to run Nextflow, which will then proceed with the managed identity for subsequent authentication.